5117_Sprint/04 - xander/src/com/company/TwuutDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.company;
+
+public class TwuutDTO {
+  private String author;
+  private String twuutText;
+
+   TwuutDTO() {
+
+  }
```

Comment:
Why do you have an empty constructor?

Reply:
There is none I think I planned to do something with it previous demo but decided not to and forgot to remove it